Makefile

```diff
@@ -164,6 +165,8 @@ remote-deps: mod-download
 		cp -r `go list -mod=mod -m -f "{{.Dir}}" github.com/projectcalico/felix`/bpf-apache bin/bpf; \
 		chmod -R +w bin/bpf; \
 		chmod +x bin/bpf/bpf-gpl/list-* bin/bpf/bpf-gpl/calculate-*; \
+		sed -i "s/amd64/arm64/" ./bin/bpf/bpf-gpl/Makefile; \
```

Comment:
Hm, I'm not sure if this is a reliable way to go about this.

I think we'd want to enhance that Makefile to have support for multi-arch in a separate PR. It lives here: https://github.com/projectcalico/felix/blob/master/bpf-gpl/Makefile

Reply:
Totally agree.